doc/syntax/literals.rdoc

```diff
@@ -310,6 +327,11 @@ You may place expressions inside the array:
   [1, 1 + 1, 1 + 2]
   [1, [1 + 1, [1 + 2]]]
 
+See also:
+
+* {%w: String-Array Literals}[#label-25w-3A+String-Array+Literals]
+* {%i: Symbol-Array Literals}[#label-25i-3A+Symbol-Array+Literals]
```

Comment:
Should mention `%W` and `%I` here, for consistency with mentioning both `%q` and `%Q` above.

Reply:
Done.